lib/api/prepareDb.js

```diff
@@ -1,24 +1,23 @@
-/*eslint-disable no-else-return */
 var
   fs = require('fs'),
   RequestObject = require('./core/models/requestObject.js'),
   InternalError = require('./core/errors/internalError'),
   async = require('async'),
   q = require('q'),
-  actualIndexes = [],
   env = {
     fixtures: 'FIXTURES',
     mappings: 'DEFAULT_MAPPING'
-  },
-  data = {};
+  };
 
 module.exports = function () {
+  this.actualIndexes = [];
```

Comment:
Not quite. I didn't think about it when I refactored this piece of code, but now that you mention it, this variable is poorly named.
It contains all known indexes at the start of Kuzzle. So it should really be named `knownIndexes`.
But now that we maintain a cache containing all known indexes to handle multi-indexes, this variable should be removed altogether, and we should use this cache instead.

Problem is: I'm currently reworking this index cache because of a bug discovered yesterday by @AnthonySendra. So I propose to keep this variable for now, and in my next PR I'll remove it completely.

What do you think about this?